### ISSUE TITLE
crop-video.py doesn't return any message after running for some videos? 

### ISSUE THREAD

**Issue by CrumpThePenguin** [AliaksandrSiarohin/first-order-model#256]
Some videos it works fine and will give me a suggested command, sometimes it won't say anything.

![image](https://user-images.githubusercontent.com/24972809/91627892-5e930300-e980-11ea-9ed4-fcb1ebff6ea8.png)

This is an example of what happens when nothing happens. I've had it work before though so I'm not sure what the problem is. I would think it would return an error or something but I'm not sure. [This](https://youtu.be/3tVrntKgdN0) is my source video, cut to only show the parts where his face is actually visible. Could it be that it's too far away or he moves around too much?

**Comment by AliaksandrSiarohin** [comment]
Yes it moves to much, you can change minimum frames to smaller number.

**Comment by ForInfinity** [comment]
Try to change the source code of "**crop-video.py**" at line 72 to
`    return f'ffmpeg -i {inp} -ss {start} -t {time} -filter:v "crop={w}:{h}:{left}:{top}, scale={scale}" crop.mp4'`

**Comment by CrumpThePenguin** [comment]
The line seems to already be exactly the same. Thank you though.
